DSCResources/MSFT_xDnsServerForwarder/MSFT_xDnsServerForwarder.psm1

```diff
@@ -0,0 +1,76 @@
+function Get-TargetResource
+{
+    [OutputType([Hashtable])]
+    param
+    (
+        [Parameter(Mandatory)]
+        [string]$IPAddress,
+        [bool]$RemoveAll = $false,
+        [ValidateSet('Present','Absent')]
+        [string]$Ensure = 'Present'
+    )
+    Write-Verbose 'Getting DNS Forwarders'
+    Write-Output @{Forwarders = (Get-CimInstance -Namespace root\MicrosoftDNS -ClassName microsoftdns_server).Forwarders}
```

Comment:
Your return from Get-TargetResource should match the schema you wrote in schema.mof